components/expenses/MarkExpenseAsPaidBtn.js

```diff
@@ -87,6 +91,8 @@ class MarkExpenseAsPaidBtn extends React.Component {
       disabledMessage = intl.formatMessage(this.messages.insufficientBalance);
     }
 
+    if (error) this.passErrorMessage;
```

Comment:
What's the effect of doing this?

Reply:
apologies, I think this was unintentionally left in from when I was trying to figure out how to pass the error to this button's parent.